time/src/scheduler.ts

```diff
@@ -1,5 +1,5 @@
 import {Listener} from 'xstream';
-const makeAccumulator = require('sorted-immutable-list').default;
+import {makeAccumulator} from 'sorted-immutable-list';
```

Comment:
Shouldnt this be `import makeAccumulator from ...`, (the default import)?

Reply:
Haha yes, saw that after running the tests!